README.md

```diff
@@ -69,3 +75,78 @@ When the Django server is running, you can start the Django shell with:
 Run the back-end test suite with:
 
     docker-compose exec backend pipenv run python manage.py test --settings=donate.settings_test
+
+
+## Basket
+
+[Basket](https://github.com/mozmeao/basket) is a tool run by MoCo to manage newsletter subscriptions and donations. It's listening for messages (JSON) sent to a SQS queue.
+
+### Basket donations queue:
+
+Basket has 4 [donation event types](https://github.com/mozmeao/basket/blob/master/basket/news/management/commands/process_donations_queue.py#L21-L26): 
+
+- `donation`: process a donation and send its data to SFDC,
+- `crm_petition_data`: add petition signature to SFDC,
+- `newsletter_signup_data`: newsletter signup for the foundation site,
+- `DEFAULT`: process a followup Stripe event on a donation.
+
+For this project, we're only using the `donation` event type.
+
+### Donation event type
+
+Example of a donation message sent to Basket:
```

Comment:
```suggestion
Example of a donation message sent to Basket, via SQS as that's known to work right now and a donation endpoint isn't yet built into Basket:
```

Reply:
I wouldn't add that: Pmac was very elusive when I ask them if they plan to rewrite that component. But let's add more content about SQS and how that work.

Reply:
```suggestion
Example of a donation message sent to Basket, via SQS:
```

Reply:
🕵️‍♂️